src/Seq.App.Opsgenie/OpsgenieApiClient.cs

```diff
@@ -6,15 +6,15 @@
 using System.Text.Json;
 using System.Text.Json.Serialization;
 using System.Threading.Tasks;
-using Seq.Apps;
+using Seq.App.Opsgenie.Classes;
+using Seq.App.Opsgenie.Interfaces;
 
 namespace Seq.App.Opsgenie
 {
     class OpsgenieApiClient : IOpsgenieApiClient, IDisposable
```

Comment:
This type could also move to `Seq.App.Opsgenie.Client`

Reply:
Reorganised into Api and Client